Tools/MiniBrowser/MiniBrowser.xcodeproj/project.pbxproj

```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 46;
+	objectVersion = 52;
```

Comment:
Please revert this change.

Reply:
I'm curious, why did this change happen in the first place? (I didn't change it manually)

Reply:
I don't know, Xcode...

Reply:
Fixed!